pkgs/data/themes/plata/default.nix

```diff
@@ -42,13 +40,12 @@ stdenv.mkDerivation rec {
   ++ stdenv.lib.optionals mateSupport [ gtk3 marco ]
   ++ stdenv.lib.optional telegramSupport zip;
 
-  buildInputs = [
-    gdk-pixbuf
-    librsvg
```

Comment:
From reading https://gitlab.com/tista500/plata-theme#installation-from-git-source, I would expect librsvg to be needed for generating images or something. So it should probably go to `nativeBuildInputs`.

But here it looks like gdk-pixbuf is needed as a library: https://gitlab.com/tista500/plata-theme/-/blob/1dd0953f1d6a35777388d39f46e2428c038042be/configure.ac#L42

Reply:
`librsvg` is not needed or mentioned anywhere in the repo; `inkscape` is used to generate images.

`gdk-pixbuf` seems to be a sanity check, there's not really a reason to require it to build the theme. I added it to `nativeBuildInputs` just in case it causes an error.